substrate-archive/src/logger.rs

```diff
@@ -98,6 +99,7 @@ pub fn init(config: LoggerConfig) -> io::Result<()> {
 			.level_for("staking", log::LevelFilter::Warn)
 			.level_for("cranelift_codegen", log::LevelFilter::Warn)
 			.level_for("wasm-heap", log::LevelFilter::Error)
+			.level_for("regalloc", log::LevelFilter::Warn)
```

Comment:
dq: what is the `regalloc` module?

Reply:
it's some `CPU instruction register` allocation crate that is used by `wasm/wasmtime` but not sure how it's used in archive, maybe @insipx  can explain :)

Reply:
It's not used in archive at all actually. During one of the 'update dependency' PR's, something must have changed within `regalloc` itself, or maybe the wasm executor in substrate, but it was printing out basically every instruction going through the executor which made it impossible to read any other logs